docs/en/connector-v2/source/Hive.md

```diff
@@ -10,7 +10,7 @@ Read data from Hive.
 
 In order to use this connector, You must ensure your spark/flink cluster already integrated hive. The tested hive version is 2.3.9.
```

Comment:
Does this line also need to be modified？

Reply:
> Does this line also need to be modified？

Yes, I will update it in the next two days.

Reply:
I haven't tested hive version 3.1.3 on Flink and Spark, so it should remain unchanged here.